### PR TITLE
ui-fixes

### PR DESCRIPTION
**Pull request by sakinaboriwala**
### Why?

UI needs to be improved.

### This PR addresses the issues by:

1) Snackbars for error messages on Authentication Screen
2) Sign Out Button on Side Drawer
3) Customer Since Info on Account
4) Email UI, Password UI
5) Account missing fields
6) Cart Text

### CODE REVIEW

File: lib/screens/account.dart [hunk 1]
```diff
@@ -21,34 +42,58 @@ class Account extends StatelessWidget {
...
+            color: Colors.grey.shade100,
+            child: ListTile(
+              title: Text(
+                "Customer since $createdAtString",
+                style: TextStyle(
+                    color: Colors.grey,
+                    fontSize: 12,
+                    fontWeight: FontWeight.w600),
+              ),
+              onTap: () {},
```
Comment (gopalshimpi) [inline lib/screens/account.dart:72]: no method call for onTap().

File: lib/screens/account.dart [hunk 2]
```diff
@@ -21,34 +42,58 @@ class Account extends StatelessWidget {
...
-          //     style: TextStyle( ),
-          //   ),
-          //   onTap: () {},
-          // ),
+          ListTile(
+            title: Text(
+              "Addresses",
+              style: _textStyle,
+            ),
+            onTap: () {
```
Comment (gopalshimpi) [inline lib/screens/account.dart:89]: same here

**Review by gopalshimpi — CHANGES_REQUESTED** [review]
@sakina1403 Please go thorough the few questions asked.

File: lib/screens/account.dart (hunk 1 above)
Comment (sakinaboriwala) [reply lib/screens/account.dart:72]: This is just info about the user, doesn't require to lead somewhere onTap, that's why I have kept it empty.

File: lib/screens/account.dart (hunk 2 above)
Comment (sakinaboriwala) [reply lib/screens/account.dart:89]: Line 90 is inside onTap

**Review by gopalshimpi — APPROVED** [review]
LGTM 👍

**Pull request merged**